### ISSUE TITLE
bug: turning circles on private track roads coloured wrong

### ISSUE THREAD

**Issue by k-yle** [gravitystorm/openstreetmap-carto#4334]
### Expected behaviour

Turning circles on track roads with `access=private` should be the same colour as the track (`#e9dcce` if private, `#996600` if public). 

### Actual behaviour

Turning circles always use `#996600`, even if the way and node have `access=private`.

This stands out a lot and looks like a tagging error.

![image](https://user-images.githubusercontent.com/16009897/108618868-61ca0000-7486-11eb-93e0-ad97714110ba.png)

### Links and screenshots illustrating the problem

See screenshot above. Source: [here](https://osm.org/#map=18/-36.80940/174.62429), [also here](https://osm.org/#map=18/-36.80953/174.61454)

**Comment by imagico** [comment]
Note drawing the turning circle with the private access color could be confused with urban landuse fills.  But in general this indeed should be fixed in some form.

**Comment by pnorman** [comment]
I find the highway=turning_circle rendering for tracks poor in general, standing out too much. I would be in favour of removing it and only showing turning circles for roads rendered with the thick fill + thin casing as opposed to the style of tracks, cycleways, footways, etc.

**Comment by imagico** [comment]
Until #4096 we rendered highway=track polygons with a lighter brown fill - see https://github.com/gravitystorm/openstreetmap-carto/pull/4096#pullrequestreview-383532070 - that would be a suitable starting point for changing the turning circle design.

In the ac-style i use a similarly light brown fill for both turning circles and ground unit rendering ([details](http://blog.imagico.de/navigating-the-maze-part-2/)):

![ac-style](http://blog.imagico.de/wp-content/uploads/2021/05/turning_circles_track_18_40.png)

Practically turning circles for tracks are pretty common and pretty meaningful for map users since many tracks are too narrow to turn around on the way.  Providing feedback to where they are mapped and where not is highly beneficial.  Let's not further turn this into a urban focus style by further neglecting and dropping rural features.

**Comment by k-yle** [comment]
another oddity with turning circles, underground ones look weird.

<p float="left">
  <a href="https://osm.org/#map=19/-36.84234/174.75862"><img src="https://user-images.githubusercontent.com/16009897/206812061-1a59c5bb-7a5b-46d3-8b4f-546f93c5d052.png" /></a>
  <a href="https://osm.org/#map=19/-36.87432/174.74415"><img src="https://user-images.githubusercontent.com/16009897/206812077-afa51540-2cb1-418c-ade5-b282e3fd8fac.png" /></a>
</p>

**Comment by imagico** [comment]
This has nothing to do with track roads so please open a separate issue (and provide links to the examples).

(this is related to #2558 - for tunnels it is still a separate problem).

**Comment by k-yle** [comment]
okay, i've opened #4747. You can click the examples above to see the osm feature

**Comment by romwriter** [comment]
This is a duplicate of https://github.com/gravitystorm/openstreetmap-carto/issues/340, isn't it?

**Comment by imagico** [comment]
Not quite.  #340 was about access restriction dashing - and was closed because rendering such correctly on turning circles (which are currently rendered in layers separate from the linear roads and road polygons) is hard and would mean significant additional complexity.  This would only be a possibility based on a solution for #2046 - which however conflicts with #3281.

This specific problem however can be solved within the existing layer structure.
